client.go

```diff
@@ -33,30 +48,29 @@ func doClient() {
 	// client
 	client, err := rpc.Dial(*g_sock, addr)
 	if err != nil {
-		if *g_sock == "unix" && fileExists(addr) {
-			os.Remove(addr)
+		if command == "exit" {
+			log.Fatal(err)
 		}
 
+		if *g_sock == "unix" {
+			_ = os.Remove(addr)
```

Comment:
Understood.  I'm asking about why it's "_ = os.Remove(addr)" instead of just "os.Remove(addr)".